project-base/tests/App/Acceptance/acceptance/PageObject/Front/RegistrationPage.php

```diff
@@ -31,7 +32,7 @@ public function register($firstName, $lastName, $email, $firstPassword, $secondP
         );
         $frontCheckboxClicker->check();
 
-        $this->tester->wait(TimedFormTypeExtension::MINIMUM_FORM_FILLING_SECONDS);
+        $this->tester->wait(self::MINIMUM_FORM_SUBMIT_WAIT_TIME);
```

Comment:
Do we really want to keep waiting 10 seconds?

Reply:
Good thought. I removed the commit.